Formula/c/cortexso.rb

```diff
@@ -0,0 +1,34 @@
+require "language/node"
+
+class Cortexso < Formula
+  desc "Drop-in, local AI alternative to the OpenAI stack"
+  homepage "https://jan.ai/cortex"
+  url "https://github.com/janhq/cortex.git",
```

Comment:
is there any plan to publish the tarball to npmjs?

Reply:
Yeah, we will work on compiling a tarball for npmjs publishing in the next release.